test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/FileExtensionsAttributeAdapterTest.cs

```diff
@@ -15,26 +15,60 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding.Validation
     public class FileExtensionsAttributeAdapterTest
     {
         [Theory]
-        [InlineData("jpg,png")]
-        [InlineData("jpg, png")]
-        [InlineData("JPEG, Png")]
-        [ReplaceCulture]
-        public void AddValidation_WithoutLocalization(string extensions)
+        [InlineData("", "png,jpg,jpeg,gif")]
+        [InlineData(null, "png,jpg,jpeg,gif")]
```

Comment:
Include a couple of uppercase characters in this data. Should be lower-cased in the error message.